style/main.scss

```diff
@@ -0,0 +1,12 @@
+@import "header.scss" ;

```

Comment:
if the imported files are only used as partials, then append '_' as a suffix in their names to prevent their css generation separately